src/test/suite/versionUtils.test.ts

```diff
@@ -18,69 +18,101 @@
-	});
+		test("validate url for existing 1.3.2 version", async () => {
+			await validateVersion('1.3.2', Platform.LINUX, 'https://github.com/apache/camel-k/releases/download/v1.3.2/camel-k-client-1.3.2-linux-64bit.tar.gz');
+		});
 
-	test("validate url for existing 1.4.0 windows version", async () => {
-		await validateVersion('1.4.0', Platform.WINDOWS, 'https://github.com/apache/camel-k/releases/download/v1.4.0/camel-k-client-1.4.0-windows-64bit.tar.gz');
-	});
+		test("validate url for existing 1.4.0 version", async () => {
+			await validateVersion('1.4.0', Platform.LINUX, 'https://github.com/apache/camel-k/releases/download/v1.4.0/camel-k-client-1.4.0-linux-64bit.tar.gz');
+		});
 
-	test("validate url for existing 1.4.0 MacOS version", async () => {
-		await validateVersion('1.4.0', Platform.MACOS, 'https://github.com/apache/camel-k/releases/download/v1.4.0/camel-k-client-1.4.0-mac-64bit.tar.gz');
-	});
+		test("validate url for existing 1.4.0 windows version", async () => {
+			await validateVersion('1.4.0', Platform.WINDOWS, 'https://github.com/apache/camel-k/releases/download/v1.4.0/camel-k-client-1.4.0-windows-64bit.tar.gz');
+		});
 
-	test("validate invalid url for xyz1 version", async () => {
-		await invalidateVersion('xyz1', Platform.LINUX);
-	});
+		test("validate url for existing 1.4.0 MacOS version", async () => {
+			await validateVersion('1.4.0', Platform.MACOS, 'https://github.com/apache/camel-k/releases/download/v1.4.0/camel-k-client-1.4.0-mac-64bit.tar.gz');
+		});
+
+		test("validate invalid url for xyz1 version", async () => {
+			await invalidateVersion('xyz1', Platform.LINUX);
+		});
 
-	async function validateVersion(tagName: string, platform: Platform, urlToTest: string): Promise<void> {
-		try {
-			const testUrl = await versionUtils.getDownloadURLForCamelKTag(tagName, platform);
-			should.equal(testUrl, urlToTest);
-		} catch (error) {
-			should.fail(error);
+		async function validateVersion(tagName: string, platform: Platform, urlToTest: string): Promise<void> {
+			try {
+				const testUrl = await versionUtils.getDownloadURLForCamelKTag(tagName, platform);
+				should.equal(testUrl, urlToTest);
+			} catch (error) {
+				should.fail(error);
+			}
 		}
-	}
-
-	async function invalidateVersion(tagName: string, platform: Platform): Promise<void> {
-		try {
-			await versionUtils.getDownloadURLForCamelKTag(tagName, platform);
-			should.fail(`Downloading invalid Camel-K version (${tagName}) did not fail!`);
-		} catch (error) {
-			should.exist(error);
+
+		async function invalidateVersion(tagName: string, platform: Platform): Promise<void> {
+			try {
+				await versionUtils.getDownloadURLForCamelKTag(tagName, platform);
+				should.fail(`Downloading invalid Camel-K version (${tagName}) did not fail!`);
+			} catch (error) {
+				should.exist(error);
+			}
 		}
-	}
+	});
```

Comment:
all this part is just that it has been grouped inside a "sub" test suite

Reply:
I recommend to use "hide whitespace change" to hide this kind of change when reviewing this PR